titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/appscale/AWSAppAutoScalingUtil.java

```diff
@@ -19,9 +19,10 @@
 import com.netflix.titus.api.appscale.model.AutoScalableTarget;
 
 public class AWSAppAutoScalingUtil {
-    public static String buildGatewayResourceId(String jobId, String awsGatewayEndpointPrefix, String region, String stack) {
+    public static String buildGatewayResourceId(String jobId, String awsGatewayEndpointPrefix, String region,
+                                                String targetStage) {
```

Comment:
Maybe worth a comment that we expect that stage name at cell stack name to match?

Reply:
IIUC, it should be the federated stack name now, no?

Reply:
it is going to be the federated stack name, but I wanted to keep it more open w/ targetStage. I'll put a comment about it